packages/apps-config/src/ui/logos/nodes/generated/kineraPNG.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright 2017-2024 @polkadot/apps authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+// Do not edit. Auto-generated via node scripts/imgConvert.mjs
+
+export const nodesKineraPNG = 'data:image/jpeg;base64,/9j/2wCEAAgGBgcGBQgHBwcJCQgKDBQNDAsLDBkSEw8UHRofHh0aHBwgJC4nICIsIxwcKDcpLDAxNDQ0Hyc5PTgyPC4zNDIBCQkJDAsMGA0NGDIhHCEyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMjIyMv/AABEIADAAMAMBIgACEQEDEQH/xAGiAAABBQEBAQEBAQAAAAAAAAAAAQIDBAUGBwgJCgsQAAIBAwMCBAMFBQQEAAABfQECAwAEEQUSITFBBhNRYQcicRQygZGhCCNCscEVUtHwJDNicoIJChYXGBkaJSYnKCkqNDU2Nzg5OkNERUZHSElKU1RVVldYWVpjZGVmZ2hpanN0dXZ3eHl6g4SFhoeIiYqSk5SVlpeYmZqio6Slpqeoqaqys7S1tre4ubrCw8TFxsfIycrS09TV1tfY2drh4uPk5ebn6Onq8fLz9PX29/j5+gEAAwEBAQEBAQEBAQAAAAAAAAECAwQFBgcICQoLEQACAQIEBAMEBwUEBAABAncAAQIDEQQFITEGEkFRB2FxEyIygQgUQpGhscEJIzNS8BVictEKFiQ04SXxFxgZGiYnKCkqNTY3ODk6Q0RFRkdISUpTVFVWV1hZWmNkZWZnaGlqc3R1dnd4eXqCg4SFhoeIiYqSk5SVlpeYmZqio6Slpqeoqaqys7S1tre4ubrCw8TFxsfIycrS09TV1tfY2dri4+Tl5ufo6ery8/T19vf4+fr/2gAMAwEAAhEDEQA/APM7e3WeRo8PxzuAyFHfI/KuustB0mPw8moXbTR4zukPIPuo7gnpXPafbI17CJpTFEzYkcHHy9/zHFdct3ZXsssUUlxcLI5MYdTtijB5AGcYwccfpXvZl7WhNKL039D6JKSYzTpbO0he7a1lltkIVkjAZkJxg47++PWrHighLSxImkkgJVVY/dRe3T1/xqrosUjXepwRwQt+9U8gKoyTxgjgHb9aZd6lo8lxeJPDdQySJsZCcLG3GAAB2IBz05NcscfiZVFJS1Vvw/rUtVJrRMzI7wi1w+NzknHpk06yw0gwDnuT3rLimeKVXTMbDkY6itPT3YsMKME9fQV9zRcXSsjdN2Lvh/R01O3EskgCIQTlT8y85x75FXrz7DoEIitvMlecnG7gbeCfwGfzIFWVtE0nSVW2uJpQ6MPJZVYxH+9z3JHTpWZaWgvob26kZ2eINGZJR1OOuAePvdvf1r4zMMRiaknGV1H8NDmrYjliubS9l8zorG0tbbTxqULSiWULLL83LjA7HnA//XXLa1drbapfB7WJmuI12Slc7GBHzL/L/wDVWvdstpoUU0keZpY1ZJBw2cc59sHGPTisfVdCuYpbcl4mlucKvPyrjHXP1rhw3NzqUO5jLGU6VSMG7N/0znhHLuLPnnB5PJzW3p0R8r/e6E9hTbXRsxpNc3CRo0jIMKTjbn8OtbNjbw+bsRhIg/jIxj1NfX4fGU4/um/efr03O+i4y95PQ3rBbqNpGFsJGMHlksobLEcnnoeetUo9Gkt70MUcLPFtki6+Y397P9KdrXir7D4auJNOaMNuVDOr7sBuCRjoRkUut+JLq3it1tHVJhp8zq0pDurKo2nPQmvhK1evUqu+i109EcU68Urct7W/yH/ZNUtbGOMBW2oIhHNGcMoxkc/7o6VmosDybNQka7gjkw8rIRsO3gLjJ65zx2FXvB3jSyh8K2EGqXVqrohQAnJCjIG4Y6ms+z17T7i81cSmF7YztJFwdoB5zjv0Fd2Hk6fxK3oeXUxEHadJare/+fy6Dba2iuo1dJGk2sQF3dQMfrz+lbOkiBbmVNsBREBbLH5cnr7jHFcrY6vaTaBN86w3S7xCNuMA8g4HHUmqPhnVLYaZ5Usjtfk4Ugk7UHAUjGO/XNexHF+0qRg18WisXgswqTqODSUb+n3dz//Z';
```

Comment:
Are we sure the image is correct, I see a PNG file below but then the generated image is saying jpeg

Reply:
You're right, that being said we corrected the base64 of the image in question